tests/scripts/thread-cert/node.py

```diff
@@ -65,7 +65,21 @@ class OtbrDocker:
     _border_routing_counters = None
 
     def __init__(self, nodeid: int, backbone_network: str, **kwargs):
+        """
```

Comment:
Missing the title? https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

```suggestion
        """Initializes an OtbrDocker node.
```

Reply:
will update